templates/partial/flex.html

```diff
@@ -1,3 +1,74 @@
-<p>{{ _('Built with %(pelican_url)s using %(flex_url)s theme',
-  pelican_url='<a href="http://getpelican.com" target="_blank">Pelican</a>',
-  flex_url='<a href="http://bit.ly/flex-pelican" target="_blank">Flex</a>'|safe) }}</p>
+<p>
+{{
+  _('Built with %(pelican_url)s using %(flex_url)s theme',
+    pelican_url='<a href="http://getpelican.com" target="_blank">Pelican</a>',
+    flex_url='<a href="http://bit.ly/flex-pelican" target="_blank">Flex</a>'|safe)
+}}
+{% if THEME_COLOR_ENABLE_USER_OVERRIDE %}
+  <span class="footer-separator">|</span>
+  Switch to the
+  <a href="javascript:void(0)" onclick="switchTheme('dark')">
+    dark
+  </a>|<a href="javascript:void(0)" onclick="switchTheme('light')">
+    light
+  </a>
+  {% if THEME_COLOR_AUTO_DETECT_BROWSER_PREFERENCE %}
```

Comment:
I guess you don't need this, if the user can override it could choose any, or browser should not be an option here